x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -148,11 +148,21 @@ export class ObservabilityAIAssistantClient {
     persist: boolean;
     responseLanguage?: string;
     conversationId?: string;
+    isNewConversation?: boolean;
+    isPublicConversation?: boolean;
```

Comment:
I think we might only need `public`, and generate the conversationId in the function itself - WDYT?

Reply:
Not sure I get it. Generate the conversation id and pass it as context to the chat ?